backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.IdentityModel.Tokens;
 using Azure.Storage.Blobs;
+using EasyMeets.Core.WebAPI.Validators.Sample;
```

Comment:
Why did you add this?

Reply:
Fixed namespace in order to correspond to file location

Reply:
Fixed